src/turbomind/triton_backend/llama/LlamaTritonModel.cc

```diff
@@ -378,7 +378,7 @@ LlamaTritonModel<T>::createModelInstance(int
     std::unique_ptr<ft::Allocator<ft::AllocatorType::CUDA>> allocator(
         new ft::Allocator<ft::AllocatorType::CUDA>(device_id));
 
-    allocator->setStream(stream);
+    allocator->setStream(instance->allocator->returnStream());
```

Comment:
This is likely to break TIS streaming output for concurrent requests. Do we still need it? @lvhan028